test/utils/image/manifest.go

```diff
@@ -99,26 +88,26 @@ func initReg() RegistryList {
 }
 
 var (
-	registry = initReg()
+	initRegistry = RegistryList{
+		GcAuthenticatedRegistry: "gcr.io/authenticated-image-pulling",
+		E2eRegistry:             "gcr.io/kubernetes-e2e-test-images",
+		PromoterE2eRegistry:     "k8s.gcr.io/e2e-test-images",
+		BuildImageRegistry:      "k8s.gcr.io/build-image",
+		InvalidRegistry:         "invalid.com/invalid",
+		GcEtcdRegistry:          "k8s.gcr.io",
+		GcRegistry:              "k8s.gcr.io",
+		SigStorageRegistry:      "k8s.gcr.io/sig-storage",
+		PrivateRegistry:         "gcr.io/k8s-authenticated-test",
+		SampleRegistry:          "gcr.io/google-samples",
+		GcrReleaseRegistry:      "gcr.io/gke-release",
+		MicrosoftRegistry:       "mcr.microsoft.com",
+		DockerLibraryRegistry:   "docker.io/library",
+	}
 
-	// PrivateRegistry is an image repository that requires authentication
-	PrivateRegistry = registry.PrivateRegistry
+	registry = initReg()
 
 	// Preconfigured image configs
-	dockerLibraryRegistry   = "docker.io/library"
-	e2eRegistry             = registry.E2eRegistry
-	promoterE2eRegistry     = registry.PromoterE2eRegistry
-	buildImageRegistry      = registry.BuildImageRegistry
-	gcAuthenticatedRegistry = registry.GcAuthenticatedRegistry
-	gcEtcdRegistry          = registry.GcEtcdRegistry
-	gcRegistry              = registry.GcRegistry
-	sigStorageRegistry      = registry.SigStorageRegistry
-	gcrReleaseRegistry      = registry.GcrReleaseRegistry
-	invalidRegistry         = registry.InvalidRegistry
-	sampleRegistry          = registry.SampleRegistry
-	microsoftRegistry       = registry.MicrosoftRegistry
-
-	imageConfigs, originalImageConfigs = initImageConfigs()
```

Comment:
This is a section that exemplifies the cleanup:
 - most values were hardcoded above this, so just refer to that value
 - docker wasn't defined above so thats hardcoded here
 - most are unexported but PrivateRegistry is, even though its not used elsewhere.